docs/index.html

```diff
@@ -279,13 +280,39 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 						const linkElement = createLink( pageName, pageURL );
 						listElement.appendChild( linkElement );
 
+						// Searchable properties and methods
+
+						const searchable = [];
+
+						page.properties?.forEach( name => {
+
+							const linkElement = createLink( `.${name}`, `${pageURL}/.${name}` );
+							linkElement.classList.add( 'hidden' );
+							listElement.appendChild( linkElement );
+		
+
+							searchable.push( { name, linkElement } );
+		
+						} );
+
+						page.methods?.forEach( name => {
+
+							const linkElement = createLink( `.${name}()`, `${pageURL}/.${name}` );
+							linkElement.classList.add( 'hidden' );
+							listElement.appendChild( linkElement );
+
+							searchable.push( { name, linkElement } );
+		
+						} );
+
```

Comment:
Need to find a way to property link to properties and methods. Furthermore, characters like `.` and `()` seem to be removed from search results, presumably from the bold regex scrubbing them out.